users.py

```diff
@@ -0,0 +1,24 @@
+import json
+
+class User():
+    def __init__(self, email, name, contact_number, listings):
+        self.email = email
+        self.name = name
+        self.contact_number = contact_number
+        self.listings = listings
+
+    @staticmethod
+    def from_firestore(email, source):
+        return User(email, source['profile']['name'], source['profile']['contact_number'], source['listings'])
+
+    def to_firestore(self):
```

Comment:
why isn't email returned in this json?

Reply:
saving to firestore doesn't require the email, as the email is used as the firestore document ID: https://github.com/maggiekoesno/pet-adoption/pull/18/files#diff-2613ed664110f871e6fb8c6858b7d2b5R34